src/run-teamscale-upload.sh

```diff
@@ -10,9 +10,12 @@ fi
 chmod +x teamscale-upload;
 
 # mandatory arguments that have to be always present
-ARGS=( "--server" "$SERVER" "--project" "$PROJECT" "--user" "$USER" "--partition" "$PARTITION" "--accesskey" "$ACCESSKEY" "--format" "$FORMAT" )
+ARGS=( "--server" "$SERVER" "--project" "$PROJECT" "--user" "$USER" "--accesskey" "$ACCESSKEY" "--partition" "$PARTITION" )
 
 # optional parameters. We only use them if they have been set
+if [ -n "$FORMAT" ]; then
```

Comment:
AFAIK, the format is mandatory, so we should keep it mandatory here as well. Calling teamscale-upload without a format will result in an error.

Reply:
You can specify the format with the --format option, or with the --input option, or with both. I'll update the docs to make this clearer.